public/toolkit/styles/atoms/_forms.scss

```diff
@@ -133,8 +133,8 @@ label {
 
 // Additional input helper text
 .form-note {
```

Comment:
it looks like form-note is used in many places in webapp, do we know what this will look like on the webapp forms?